content/python/concepts/dates/terms/time/time.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.time()'
+Description: 'Returns the time '
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.time()` method returns the time in seconds that has passed since the epoch set on the computer.
```

Comment:
Again, is it `.time()` or `datetime.time()`. Also specify the return type and its properties.